src/docs/development/ui/layout/index.md

```diff
@@ -1083,44 +1087,46 @@ or to add borders or margins. You can change the device's background by
 placing the entire layout into a Container and changing its background color
 or image.
 
-<div class="row"> <div class="col-lg-6" markdown="1">
-
-#### Container summary:
-
-* Add padding, margins, borders
-* Change background color or image
-* Contains a single child widget, but that child can be a Row, Column,
-  or even the root of a widget tree
-
-</div> <div class="col-lg-6" markdown="1">
-
-<img src="/docs/development/ui/layout/images/margin-padding-border.png" alt="a diagram showing that margins, borders, and padding, that surround content in a container">
-
-</div> </div>
-
-#### Container examples:
+<div class="row">
+<div class="col-lg-6" markdown="1">
+  <h4 class="no_toc">Summary (Container)</h4>
+
+  * Add padding, margins, borders
+  * Change background color or image
+  * Contains a single child widget, but that child can be a Row, Column,
+    or even the root of a widget tree
+</div>
+<div class="col-lg-6 text-center">
+  {% asset ui/layout/margin-padding-border.png class="mb-4 mw-100"
+      width="230px"
+      alt="Diagram showing: margin, border, padding, and content" %}
+</div>
+</div>
+
+#### Examples (Container)
```

Comment:
I'm not crazy about using parens in headings. And actually, I don't know that these headings are really necessary. The sections are so short, I think a one-sentence paragraph could replace them. (It'd also fix the space/convention issues of having bullets or images right after headings. And it'd help us improve the prose in the lists.)

But... it might require more writing/time than we have right now. And your fix is certainly better than what's in the page right now.

Let's go with your paren'd headings, and maybe I'll fix them later. :)

----------------------

For the record, here's what I was thinking of doing for Container (with something similar for GridView, ListView, etc.):

### Container

[Existing paragraph about Container.]

With a Container, you can do the following:

* Add padding, margins, borders.
* Change the background color or image.
* Have only one child, but that child...

Many examples in this tutorial use Container. You can also find more Container examples in the Flutter Gallery.

The following example uses Containers to implement a layout consisting of ...